inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/internal/JdbcBatchingOutputFormat.java

```diff
@@ -122,7 +122,7 @@ public void open(int taskNumber, int numTasks) throws IOException {
         this.runtimeContext = getRuntimeContext();
         metricData = new MetricData(runtimeContext.getMetricGroup());
         if (inLongMetric != null && !inLongMetric.isEmpty()) {
```

Comment:
use SinkMetricData of connector-base after git rebase code

Reply:
All pr about metrics will be refactored in One PR in the future